doc/source/data/transforming-data.rst

```diff
@@ -312,6 +312,27 @@ To transform data with a Python class, complete these steps:
 
             ds.materialize()
 
+Avoiding out-of-memory errors
+=============================
+
+If your user defined function uses lots of memory, you might encounter out-of-memory 
+errors. To avoid these errors, configure the ``memory`` parameter:
```

Comment:
Can you also say what this will do? if you configure the memory parameter, then Ray Data will ...

Reply:
How's this?

> To avoid these errors, configure the ``memory`` parameter. It tells Ray how much memory your function uses, and prevents Ray from scheduling too many tasks on a node.

Reply:
sg